bin/oracle.bash

```diff
@@ -43,10 +43,8 @@ function current_releases {
 		arch=$(cut -f2 -d, <<<"$param")
 		local ext_list
 		ext_list=$(cut -f3 -d, <<<"$param")
-		local exts
-		exts=$(cut -f3 -d: <<<"$ext_list")
```

Comment:
Since it was cutting the 3rd item in the ext list only windows/zip and linux/x64/tar.gz were discovered.